index.js

```diff
@@ -14,6 +37,18 @@ module.exports = () => {
 			childProcess.execFileSync('rundll32.exe', ['user32.dll,LockWorkStation']);
 			break;
 		}
+		case 'linux': {
+			// See: https://askubuntu.com/questions/184728/how-do-i-lock-the-screen-from-a-terminal
+			const existingCommand = getExistingLinuxCommand();
+
+			if (existingCommand) {
+				childProcess.execFileSync(existingCommand.name, [existingCommand.arg]);
+			} else {
+				throw new Error('No applicable command found. Please consider installing xdg-screensaver, gnome-screensaver or vlock and try again.');
```

Comment:
`gnome-screensaver-command` is part of [gnome-screensaver](https://github.com/GNOME/gnome-screensaver) package, so it would require more processing than simply combining the names I'm afraid.